### ISSUE TITLE
Tooltip not rendering for any graph

### ISSUE THREAD

**Issue by NotArpit** [plouc/nivo#1823]
Hey !

I have been playing around with line charts from nivo and for some strange reason no tooltips have been getting rendered even after I have directly copied examples from the documentation ?

It seems as though this is a bug, I have re-read the documentation multiple times and could not find anything that would indicate that there is an issue with my dependencies (I have made sure peer dependencies are up to date). Furthermore this issue seems to be occurring across different browsers.

For some more insights into my project, the versioning details of my package.json are as follows:

```
"react": "^16.14.0",
"react-dom": "^16.14.0",
"react-scripts": "^3.4.4",
"@nivo/core": "^0.74.0",
"@nivo/line": "^0.74.0",
"@nivo/tooltip": "^0.74.0",
```

I'd be grateful for any clarification that can be provided

**Comment by onionhammer** [comment]
I'm seeing the same issue with funnel

**Comment by move02** [comment]
same issue on heatmap..

`   "@nivo/bar": "^0.74.0",
    "@nivo/bump": "^0.74.0",
    "@nivo/core": "^0.74.0",
    "@nivo/geo": "^0.74.1",
    "@nivo/heatmap": "^0.74.0",
    "@nivo/line": "^0.74.0",
    "@nivo/pie": "^0.74.0",
`

**Comment by SzymonKwasek** [comment]
I have the same issue on pie chart, copied exactly the code from docs and the tooltip is not there.
`    "@nivo/bar": "^0.73.1",
    "@nivo/core": "^0.78.0",
    "@nivo/line": "^0.73.0",
    "@nivo/pie": "^0.78.0",`

**Comment by plouc** [comment]
@SzymonKwasek, you should align the versions of the nivo packages.

**Comment by plouc** [comment]
The solution should be to make sure all the nivo package versions are aligned, I never ran into this issue, as it seems to solve the problem, I'm going to close this for now.

If someone still encounter this, and can reproduce it in codesandbox for investigation, I will reopen it though.

**Comment by neemanjabu** [comment]
Had the same issue.
Solved it by aligning versions to 0.79.
```
@nivo/core@0.79
@nivo/bar@0.79
@nivo/sunburst@0.79
```
